components/schemes/Typography/tests/unit/MDCTypographySchemeTests.m

```diff
@@ -149,4 +149,47 @@ - (void)testTypographyScheme201902ReturnsDefaultSizedFonts {
       mdc_isSimplyEqual:[scheme201902.overline mdc_scaledFontAtDefaultSize]]);
 }
 
+- (void)testTypographyScheme201804DisableUseCurrentContentSizeCategoryWhenAppliedByDefault {
```

Comment:
Should we add documentation to these method?

Reply:
Generally-speaking unit tests shouldn't need docs if the methods are self-documenting. Do you feel these methods could use clarification?